rclcpp/src/rclcpp/executor.cpp

```diff
@@ -268,7 +268,10 @@ Executor::add_node(rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_pt
   // If the node already has an executor
   std::atomic_bool & has_executor = node_ptr->get_associated_with_executor_atomic();
   if (has_executor.exchange(true)) {
-    throw std::runtime_error("Node has already been added to an executor.");
+    throw std::runtime_error(
+            std::string(
+              "Node (%s) has already been added to an executor.",
+              node_ptr->get_fully_qualified_name()));
```

Comment:
This is mixing printf-style string in with normal strings, and is causing the tests to fail.  I think this wants to be something like:

```suggestion
            std::string("Node '") + node_ptr->get_fully_qualified_name() + "' has already been added to an executor.");
```

(that particular line will probably fail linting, so you'll also have to fix that up)